docs/getting_started/install_on_ibexa_cloud.md

```diff
@@ -83,3 +88,13 @@ git push -u <platform.sh-remote> master
 !!! note
 
     `master` is the Platform.sh name for the production branch.
+
+!!! caution
+
+    If you want to use the [Platform.sh CLI](https://docs.platform.sh/development/cli.html), you have to set it up for Ibexa Cloud.
+    Consider using the following alias `ibexa_cloud` with the right set up instead of `platform`:
```

Comment:
```suggestion
    Consider using the alias `ibexa_cloud` with the following set up instead of `platform`:
```
For me this sounds better.

Reply:
It depends on the object we want to emphasize :)

Reply:
I don't like the word "right"…

@reithor @juskora Maybe something like the following?
```suggestion
    Consider using the following `ibexa_cloud` alias with a dedicated set up instead of `platform`:
```